src/main/java/cz/cvut/kbss/termit/dto/ConfigurationDto.java

```diff
@@ -1,19 +1,51 @@
 package cz.cvut.kbss.termit.dto;
 
+import cz.cvut.kbss.jopa.model.annotations.Id;
+import cz.cvut.kbss.jopa.model.annotations.OWLAnnotationProperty;
+import cz.cvut.kbss.jopa.model.annotations.OWLClass;
+import cz.cvut.kbss.jopa.model.annotations.OWLObjectProperty;
+import cz.cvut.kbss.termit.model.UserRole;
+import cz.cvut.kbss.termit.util.Vocabulary;
 import java.io.Serializable;
+import java.net.URL;
+import java.util.Set;
 
 /**
  * Represents configuration data provided by the server to client.
  */
+@OWLClass(iri = Vocabulary.s_c_konfigurace)
 public class ConfigurationDto implements Serializable {
 
+    @Id
+    private URL id;
+
+    @OWLAnnotationProperty(iri = Vocabulary.s_p_language)
```

Comment:
JOPA has dedicated constants for DC (among other well-known vocabularies). It would be probably better to use it here.